consensus/tendermint/tendermint.go

```diff
@@ -111,6 +111,16 @@ func NewTendermint(moduleConfig *config.ModuleConfig,
 		path.Join(moduleConfig.RootDir,
 			moduleConfig.Config.GetString("private_validator_file")))
 
+	// TODO: [Silas] we want to something better than this like not not have it in
+	// the config at all, but for now I think it's much safer to make sure we are
+	// not running the tendermint RPC as it could lead to unexpected behaviour,
+	// not least if we accidentally try to run it on the same address as our own
+	if tmintConfig.GetString("rpc_laddr") != "" {
```

Comment:
brilliant, but Im not sure that it would work;  I would almost default to setting the tendermint rpc to a different port by default `0.0.0.0:48857` eg